ts/navigation/AppStackNavigator.tsx

```diff
@@ -108,11 +102,7 @@ const InnerNavigationContainer = (props: InnerNavigationContainerProps) => {
             [MESSAGES_ROUTES.MESSAGES_HOME]: "messages",
             [ROUTES.WALLET_HOME]: "wallet",
             [SERVICES_ROUTES.SERVICES_HOME]: "services",
-            // [ROUTES.BARCODE_SCAN]: "scan",
-            ...(isNewWalletSectionEnabled
-              ? { [ROUTES.PAYMENTS_HOME]: "payments" }
-              : {}),
-            [ROUTES.PROFILE_MAIN]: "profile"
```

Comment:
Thank you for your comment! 
Actually, that route does not work to date because ‘ROUTES.PROFILE_MAIN’ no longer exists (it was a detail that was not considered when we thought of the profile move). It would be a question of clarifying if it needs to be added back in and how it should be handled. Anyway, I will talk to the team about it. 
Compared to this PR it is better to descop this issue!